### PR TITLE
Wallet detail transaction

### PR DESCRIPTION
**Pull request by jarindr**
Issue/Task Number: #947 

# Overview

Add wallet transactions tab.

# Changes

- Add wallet transactions tab.

Go to wallet details, should show a new design.

How can this change be deployed? Any special requirement?

- Deploy as usual.

### CODE REVIEW

**Comment by nicholasmueller** [comment]
change base branch, cant review 6k lines of code :p

**Comment by jarindr** [comment]
@nicholasmueller changed the branch.

File: apps/frontend/assets/src/omg-uikit/tabButton/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+
+const TabButton = styled.button`
```
Comment (nicholasmueller) [inline apps/frontend/assets/src/omg-uikit/tabButton/index.js]: I created `Tag` component already, can check on Keys page

File: apps/frontend/assets/src/omg-uikit/index.js [hunk 2]
```diff
@@ -37,10 +36,9 @@ export {
   Select,
   Switch,
   Error,
-  DetailRow,
   Tag,
   Tooltip,
-  NavCard,
```
Comment (nicholasmueller) [inline apps/frontend/assets/src/omg-uikit/index.js]: navcard is used on the keys detail page

**Comment by jarindr** [comment]
@nicholasmueller fixed.

**Pull request merged**